contrib/examples/actions/workflows/mistral-yaql-st2kv-user-scope.yaml

```diff
@@ -0,0 +1,12 @@
+version: '2.0'
+
+examples.mistral-yaql-st2kv-user-scope:
+    vars:
+        polo: unspecified
+    tasks:
+        task1:
+            action: std.noop
+            publish:
+                polo: <% st2kv('marco') %>
```

Comment:
So, I tested this st2kv.get quickly.  To be exact, because of the `()`. YAQL evaluated st2kv as an object and thinks there's a get method for st2kv object. The exact error is "No method get for receiver st2kv matches supplied arguments". Since only pluggable YAQL function is supported, I don't see an option where I can register st2kv as a module in YAQL.